ci/tasks/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+cf api $CF_API
+cf auth
+
+cf t -o $CF_ORG -s $CF_SPACE
+
+cf push $CF_APP_NAME \
```

Comment:
Does strategy rolling matter for a task only deployment?

Reply:
This application has 0 instances, so i don't think so. We could try to test to see?

Reply:
If it's not broken don't fix it and it was more of a thought.